core/src/main/resources/schema/app/20180823094346_BUG-109647_remove_publicinaccount.sql

```diff
@@ -1,44 +1,5 @@
 -- // BUG-109647_remove_publicinaccount
 -- Migration SQL that makes the change goes here.
 
-ALTER TABLE blueprint DROP COLUMN IF EXISTS publicinaccount;
```

Comment:
We shouldn't edit an SQL file

Reply:
No, this columns shouldn't have been dropped from the database in the first place. We are lucky it's still just on an RC branch.

Reply:
I've understood the problem and it is reasonable. I think you should add a comment about the problem.